### ISSUE TITLE
Status code of 404 when installing ember-cli-chosen 

### ISSUE THREAD

**Issue by atremblay-seniorlink** [green-arrow/ember-cli-chosen#28]
# Using ember 2.4.2, both `ember install:addon ember-cli-chosen` and `ember install ember-cli-chosen` return the following output:

```
Installed packages for tooling via npm.
installing ember-cli-chosen
  install bower package chosen=https://github.com/harvesthq/chosen/releases/download/v1.3.0/chosen_v1.3.0.zip
Installing browser packages via Bower...
  not-cached https://github.com/harvesthq/chosen/releases/download/v1.3.0/chosen_v1.3.0.zip=chosen=https://github.com/harvesthq/chosen/releases/download/v1.3.0/chosen_v1.3.0.zip#*
Status code of 404
Error: Status code of 404
    at createError (/path/to/project/node_modules/bower/lib/util/createError.js:4:15)
    at Request.<anonymous> (/path/to/project/node_modules/bower/lib/util/download.js:91:36)
    at emitOne (events.js:95:20)
    at Request.emit (events.js:182:7)
    at Request.onRequestResponse (/path/to/project/node_modules/bower/lib/node_modules/request/request.js:951:10)
    at emitOne (events.js:90:13)
    at ClientRequest.emit (events.js:182:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:425:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:89:23)
    at TLSSocket.socketOnData (_http_client.js:315:20)
```

bower: v1.7.7
node: 5.6.0
os: darwin x64

Any clues on why this happened or how to fix it? Clearing the bower cache and trying to install the specific version of chosen manually via bower does not work (the installation through bower will succeed, but the ember install:addon call will still throw the same error).

**Comment by dillonwelch** [comment]
I get the same error 😢

**Comment by green-arrow** [comment]
Sorry for the delayed response. I will be looking into this tonight. It looks like the URL bower is using is incorrect, appending a bunch of stuff after the actual zip file.

**Comment by green-arrow** [comment]
@atremblay-seniorlink @oniofchaos - This issue has been fixed as of v0.0.5 ([commit here](https://github.com/green-arrow/ember-cli-chosen/commit/2cc63d7d69a99c1444a6683e7d587a0b47e87816))
